searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java

```diff
@@ -0,0 +1,33 @@
+package org.jalau.at18.searchobject.model.objectrecognizer.recognizer;
+
+import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.MatchInfo;
+import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.rcnn.RcnnDetection;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RCNN implements ModelRecognizer {
+    RcnnDetection object;
+    @Override
+    public List<MatchInfo> matching(Path pathFolder, String searchCriteria, int occurrencyPercentage) {
+        List<MatchInfo> matches = new ArrayList<>();
+        try {
+            File file = pathFolder.toFile();
+            String[] listImg = file.list();
+            if (listImg == null || listImg.length == 0) {
+                System.out.println("There are no elements inside the current folder");
```

Comment:
Please remove this `System.out.println()`

Reply:
I have removed the System.out and replaced it with a logger

Reply:
All images in uploads were erased